docs/user_guide/transformation/ArcsinTransformer.rst

```diff
@@ -0,0 +1,85 @@
+.. _arcsin:
+
+.. currentmodule:: feature_engine.transformation
+
+ArcsinTransformer
+=================
+
+The :class:`ArcsinTransformer()` applies the arcsin transformation to
+numerical variables.
+
+The :class:`ArcsinTransformer()` only works with numerical variables with values between 0 and +1. If the variable contains a value outside of this range, the transformer will raise an error.
+
+Let's load the breast cancer dataset and  separate it into train and test sets (more details about the dataset :ref:`here <datasets>`).
```

Comment:
please remove this bit of the sentence:
(more details about the dataset :ref:`here <datasets>`)